tmva/sofie/src/SOFIE_common.cxx

```diff
@@ -135,14 +135,65 @@ T* UTILITY::Unidirectional_broadcast(const T* original_data, const std::vector<s
       return new_datavector;
 }
 
+template <typename T>
```

Comment:
You are missing to return the new shape from the function. 
You should also remove to pass original_data that is not used. 
This should fix the warnings !

Reply:
Done!